recipes/hermes-simulator/LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
You will need to put this file https://github.com/Josvth/hermes-simulator/blob/master/LICENSE here